source/languages/en/riakcs/cookbooks/configuration/Configuring-Riak-CS.md

```diff
@@ -70,68 +78,71 @@ the local host. If your Riak CS system has multiple nodes, however, you
 must specify the IP address and port for the Stanchion node and whether
 or not SSL is enabled.
 
-The Stanchion settings reside in the Riak CS `app.config` file, which is
-located in the `/etc` directory of each Riak CS node. The settings
-appear in the `riak_cs` config section of the file.
+The Stanchion settings reside in the Riak CS `riak-cs.conf` file, which is
+located in the `/etc/riak-cs` directory of each Riak CS node.
 
-To set the host and port for Stanchion, do the following:
+To set the host and port for Stanchion, make sure the following parameter is set
+to the host and port used by Stanchion:
 
-* For the `stanchion_ip` setting, replace `127.0.0.1` with the address
-  of the Stanchion node
-* For the `stanchion_port` setting, replace `8085` with the port for the
-  node
+* `stanchion_host` --- replace `127.0.0.1:8085` with the IP address and port
+  number of the Stanchion node
 
 ## Enabling SSL
 
-SSL is disabled by default in Stanchion, i.e. the `stanchion_ssl`
-variable is set to `false`. If Stanchion is configured to use SSL,
-change this variable to `true`. The following example configuration
-would set the Stanchion host to `localhost`, the port to `8085` (the
-default), and set up Stanchion to use SSL:
+SSL is disabled by default in Stanchion, i.e. the `stanchion_ssl` variable is
+set to `off`. If Stanchion is configured to use SSL, change this variable to
+`on`. The following example configuration would set the Stanchion host to
+`localhost`, the port to `8085` (the default), and set up Stanchion to use SSL:
+
+```riakcsconf
+stanchion_host = 127.0.0.1:8085
+stanchion_ssl = on
+```
 
-```appconfig
+```advancedconfig
 {riak_cs, [
     %% Other configs
-
-    {stanchion_ip, "127.0.0.1"},
-    {stanchion_host, 8085},
+    {stanchion_host, {"127.0.0.1", 8085}}
```

Comment:
Oh, you need comma here as there's a next line. Should out to erlang term format.